### ISSUE TITLE
A newbie friendly how-to getting started guide

### ISSUE THREAD

**Issue by zengland78** [oslabs-beta/navigate#120]
I am very interested in this project and trying it out.
However, getting started was a bit challenging and I could see for others it might be even more challenging.
A simple step-by-step guide from cloning the repo, building the docker image, running the docker image, how to start up the interface, how to point to the cluster or yaml configuration would be SUPER helpful for that.

If you gave me your "official" steps, I would even be willing to add it to your README, or other help file myself.

**Comment by brkg** [comment]
Hey @zengland78, thanks for the heads up - you’re totally right, that’s missing from our readme. I’ll add a step-by-step guide tomorrow night; would you mind if I reach out with a draft for feedback?

**Comment by zengland78** [comment]
@brkg Yes, that is fine, I will absolutely give feedback on the draft.

**Comment by hemwatie** [comment]
Hey @zengland78, thanks for reaching out, much appreciated! Let us know what you think of this first draft:

Here are the steps for getting started after cloning:
1. Run `npm  install`, and `npm run both`.
2. From there, the dev server should launch, and you will be prompted to upload your Kubernetes yaml config files. 

  -  The app will display your cluster in a network graph without your Kubernetes cluster running. However, to see live deployment and pod log data, your cluster needs to be spun up. (At this point, only deployments and pods objects can be interacted with).
  -  If there are network policy objects within your files, the top left network policy tab will provide a dropdown menu so you can inspect each policy.

Currently, we are working on packaging and distributing our app cross-platform via electron.  Right now the alpha version for macOS is out, which you can find here on our [website](https://navigates.io).

**Comment by brkg** [comment]
> 
> 
> @brkg Yes, that is fine, I will absolutely give feedback on the draft.

Added our draft to the readme, let me know what you think. I'll close this out in a couple days unless you have specific comments - otherwise, thanks for pointing this out 👍

**Comment by zengland78** [comment]
I tried it on a couple of environments yesterday and hit a different issue on each.
I plan to try again on Monday and will also review the README changes.

**Comment by zengland78** [comment]
Hello @brkg
As far as the readme goes, a prerequisites section would be nice (node, npm, gtk if on Linux, etc.)

I did run into the same type of error both attempting to run directly on Windows and Powershell, as well as on Windows WSL 2 with Ubuntu and an xserver via vcxsrv.
I can open a different issue for the problems I encountered if you like.

**Comment by brkg** [comment]
> 
> 
> Hello @brkg As far as the readme goes, a prerequisites section would be nice (node, npm, gtk if on Linux, etc.)
> 
> I did run into the same type of error both attempting to run directly on Windows and Powershell, as well as on Windows WSL 2 with Ubuntu and an xserver via vcxsrv. I can open a different issue for the problems I encountered if you like.

Great notes, thanks. I really appreciate these, and please do open any issues you see - I fixed that one you noted yesterday.
